lib/tokens/token.js

```diff
@@ -40,28 +40,34 @@ module.exports = function (log, random, P, hkdf, Bundle, error) {
     this.algorithm = 'sha256'
     this.uid = details.uid || null
     this.lifetime = details.lifetime || Infinity
-    this.createdAt = optionallyOverrideCreatedAt(details.createdAt)
+    this.createdAt = details.createdAt || 0
```

Comment:
It turns out that a bunch of test-client stuff calls these without specifying `createdAt`, e.g.:

  https://github.com/mozilla/fxa-auth-server/blob/master/test/client/api.js#L153

I'm going to revert this to defaulting to zero, which is the "maximally safe" default value because it makes them expire the soonest.